src/gql/GQLWrapper.tsx

```diff
@@ -85,6 +86,18 @@ const authenticateIfSuccessfulLink = (dispatch: Dispatch) =>
     });
   });
 
+const retryLink = new RetryLink({
+  delay: {
+    initial: 300,
+    max: Infinity,
+    jitter: true
+  },
+  attempts: {
+    max: 5,
+    retryIf: (error): boolean => error && error.response.status !== 401
```

Comment:
I think we want to retry for anything >=500. 4xx implies that the client did something wrong so that retrying won't help